content/kubernetes-backing-services/monitoring/prometheus-alerts-grafana.md

```diff
@@ -1,5 +1,68 @@
 ---
 title: "Prometheus, Alerts & Grafana"
-description: ""
-draft: true
+description: "Prometheus is monitoring system and time-series database that works together with Grafana and Alert Manage to provide an end-to-end monitoring platform for Kubernetes."
 ---
+ [Prometheus](https://prometheus.io) is at the heart of the monitoring infrastructure. It works in tandem with [Alert Manager](https://prometheus.io/docs/alerting/alertmanager/), [Grafana](https://grafana.com/) and wide list of [Exporters](https://prometheus.io/docs/instrumenting/exporters/) that provide the metrics for instrumentation.
+
+{{< img src="/assets/324asd-Prometheus_architecture.png" title="Architecture" >}}
+
+# Prometheus
+
+Prometheus is monitoring system and time-series database.
+
+It is responsible for several things:
+* Scrapping metrics data from exporters
+* Storing metrics data
+* Firing alerts
+* Response to queries
```

Comment:
Can you explain what this is or reword it? "response to queries" - I don't know what it refers to

Reply:
You can select metrics data by sending http request with query